news/2018-10-10-beatmap-spotlights-summer-2018.md

```diff
@@ -0,0 +1,309 @@
+
+Characterized by the fierce Slider Velocity changes and varied patterns with barely no rest at a high BPM of 190, this map challenges the players' readability of Slider Velocities as well as their stamina. [Mage](https://osu.ppy.sh/users/5527957)'s guest difficulties are also the highlights of this mapset. At the same time when keeping a good spread, he still tried to utilize some great concepts to pose some challenges on advancing players. 
+
+All in all, this mapset would be among good choices of practicing stamina and challenging yourself!
+
+[![](/wiki/shared/news/2018-10-10-beatmap-spotlights-summer-2018/mania/tsukidokei.jpg)](http://osu.ppy.sh/beatmapsets/678723)
+
+[FAMoss](https://osu.ppy.sh/users/7707789) presents a very unique approach to ZUN's "Lunar Clock ~ Luna Dial" with this mapset. The first track from one of the Touhou games to have been ranked in mania 4K, the musical style has allowed for fun minijacks throughout the map, and mappers can learn from the spectacular usage of the positioning of the notes that represents the music very well.
+
+[![](/wiki/shared/news/2018-10-10-beatmap-spotlights-summer-2018/mania/redemption.jpg)](http://osu.ppy.sh/beatmapsets/788366)
+
+Another one of [error_exe777](https://osu.ppy.sh/users/7975796)'s Slider Velocity masterpieces, "Tetsuo's Redemption" by KOAN Sound & Asa is an enjoyable, chill-enducing marathon map with unpredictable, erratic speed changes that require the player to have a complete understanding of the music. 
+
+Its use of varying concepts, particularly the repetition of specific patterns, is something that many mappers should keep in mind when making maps with Slider Velocity changes.
+
+[![](/wiki/shared/news/2018-10-10-beatmap-spotlights-summer-2018/mania/volcanic.jpg)](http://osu.ppy.sh/beatmapsets/765056)
+
+[_underjoy](https://osu.ppy.sh/users/2235750) strikes again with his new extreme 7k chart and keeps the ranked section alive for top-tier players again. This marathon contains many different skillset, all at extremely high levels, including stamina, heavy chordstreaming and jack speed. As seen by the scoreboard, even top-tier players have to put high effort to play this map well. 
+
+The map includes not only challenge element, but also consistency and good pattern choices that represent the music at the proper level. This map can fairly be named as one of the hardest ranked marathons.
+
+--------
+
+We have decided to make the spreadsheet available for everyone who is interested to take a look at how we selected these mapsets. Do not be afraid to take a look at it [here](link)!
+
+We hope you enjoy our selection for this season and look forward to what we will bring to you in the future.
+
+Until then, happy clicking, drumming, catching, and hitting the piano notes! Do not forget to have a rest occasionally!
```

Comment:
the list of actions for each mode is inconsistent in style so it sounds kinda weird when u read it

maybe like... "circle clicking, drum bashing, fruit catching, and key smashing"